test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ApplicationInsights/ApplicationInsightsEndToEndTests.cs

```diff
@@ -98,7 +98,7 @@ private IHost ConfigureHost(LogLevel minLevel = LogLevel.Information, HttpAutoCo
                 })
                 .ConfigureServices(services =>
                 {
-                    ServiceDescriptor quickPulse = services.Single(s => s.ImplementationType == typeof(QuickPulseTelemetryModule));
+                    ServiceDescriptor quickPulse = services.Single(s => s.ServiceType.Name == nameof(QuickPulseTelemetryModule));
```

Comment:
I couldn't find a way to add the 3 AppInsights singletons with an ImplementationType and I don't think I fully understand the ramifications of adding them without one. This is the only thing I've found in the code that was counting on these modules having an ImplementationType, but I'm not sure if there are others.